lib/pages/movie_list_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_project/Widgets/movie_builder_json.dart';
+import 'package:flutter_project/utils/styles.dart';
+import '../Widgets/movie_information_from_movie_list.dart';
+import '../models/movie.dart';
+
+class MovieListPage extends StatelessWidget {
+  MovieListPage({super.key});
+
+  late Future<List<Movie>> _movies;
+  static const Color cardColor = Color(0xff2B1C2B);
+
+  @override
+  Widget build(BuildContext context) {
+    _movies = MovieBuilderJson().loadMoviesFromJson();
```

Comment:
We can add some Flutter best practices here. Please read this:
https://api.flutter.dev/flutter/widgets/FutureBuilder-class.html

> If the future is created at the same time as the FutureBuilder, then every time the FutureBuilder's parent is rebuilt, the asynchronous task will be restarted. A general guideline is to assume that every build method could get called every frame, and to treat omitted calls as an optimization.

TLDR, you could make this widget Stateful, declare a Future<T> movies at the beginning, init that future in the initState method, and use that future in the FutureBuilder. Like here:
https://stackoverflow.com/a/68966640

Reply:
Done